src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -4390,14 +4391,8 @@ window.bp = window.bp || {};
 
 						// Display a successful feedback if the acticity is not consistent with the displayed stream.
 						if ( ! toPrepend ) {
-							self.views.add(
-								new bp.Views.activityFeedback(
-									{
-										value: response.message,
-										type: 'updated'
-									}
-								)
-							);
```

Comment:
@amincdev Why we are not using this method and adding a HTML? Can't we add new params in the new bp.Views.activityFeedback and make them as we need.

Reply:
Resolved now.